WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/logic/ui/TimeSynchronityChecker.java

```diff
@@ -4,11 +4,13 @@
 import java.util.Timer;
 import java.util.TimerTask;
 
+import nl.tudelft.watchdog.core.logic.ui.RegularCheckerBase;
 import nl.tudelft.watchdog.logic.interval.IntervalManager;
-import nl.tudelft.watchdog.logic.ui.events.WatchDogEvent;
-import nl.tudelft.watchdog.logic.ui.events.WatchDogEvent.EventType;
-import nl.tudelft.watchdog.util.WatchDogGlobals;
-import nl.tudelft.watchdog.util.WatchDogLogger;
+import nl.tudelft.watchdog.ui.preferences.Preferences;
+import nl.tudelft.watchdog.core.logic.ui.events.WatchDogEvent;
+import nl.tudelft.watchdog.core.logic.ui.events.WatchDogEvent.EventType;
+import nl.tudelft.watchdog.core.util.WatchDogGlobals;
+import nl.tudelft.watchdog.core.util.WatchDogLogger;
 
```

Comment:
i have the feeling we can extract a lot functionality from here to the core, but please keep that in mind for later.
 Add a line note